<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GalSequenceWidget</class>
 <widget class="QWidget" name="GalSequenceWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>562</width>
    <height>326</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sequences</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QComboBox" name="Sequence_comboBox">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>0</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Select known Sequence by name</string>
       </property>
       <item>
        <property name="text">
         <string>Single Pulse</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Double Pulse</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Pulse Sequence</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="SequenceIDSpinBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>unique id of selected sequence</string>
       </property>
       <property name="readOnly">
        <bool>false</bool>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::NoButtons</enum>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="SequenceNewButton">
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Create new Sequence&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../galapicons.qrc">
         <normaloff>:/icons/filenew.png</normaloff>:/icons/filenew.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="SequenceEditButton">
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Edit command code of selected sequence&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../galapicons.qrc">
         <normaloff>:/icons/analysiswin.png</normaloff>:/icons/analysiswin.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="SequenceDeleteButton">
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Remove selected sequence from setup&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../galapicons.qrc">
         <normaloff>:/icons/delete.png</normaloff>:/icons/delete.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="SequenceLoadButton">
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Load sequence code from file&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../galapicons.qrc">
         <normaloff>:/icons/open.png</normaloff>:/icons/open.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="SequenceSaveButton">
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Export current sequence code to file&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../galapicons.qrc">
         <normaloff>:/icons/filesave.png</normaloff>:/icons/filesave.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="SequenceEditCancelButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Cancel edited code for selected sequence (restore last setup)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../galapicons.qrc">
         <normaloff>:/icons/undo.png</normaloff>:/icons/undo.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="SequenceApplyButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Apply command code for selected sequence&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../galapicons.qrc">
         <normaloff>:/icons/OK16.png</normaloff>:/icons/OK16.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QPlainTextEdit" name="SequenceTextEdit">
     <property name="toolTip">
      <string>Command code of selected sequence.</string>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../galapicons.qrc"/>
 </resources>
 <connections/>
</ui>
